app/model/commands/PublishAtomCommand.scala

```diff
@@ -99,9 +99,19 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
   }
 
   private def updateYouTube(previewAtom: MediaAtom, asset: Asset): Future[MediaAtom] = {
-    updateYoutubeMetadata(previewAtom, asset)
-    updateYoutubeThumbnail(previewAtom, asset)
-    createOrUpdateYoutubeClaim(previewAtom, asset)
+    previewAtom.channelId match {
+      case Some(channel) if youTube.allowedChannels.contains(channel) =>
```

Comment:
Just gave it a go and I think it would be rather confusing because the initial headline and description are not imported when you manually add the asset.

I agree, ship this and see. We may need to do a bit more additional work since selecting a YouTube channel is mandatory right now, which blocks this. That won't be true once #507 is merged